irods/files/rule-bases/ipc-housekeeping.re

```diff
@@ -130,16 +130,54 @@ ipc_rescheduleStorageFreeSpaceDetermination {
 
 _ipc_rmTrash {
   writeLine('serverLog', 'DS: starting trash removal');
+  *zone = ipc_ZONE;
+  *verdict = true;
+  msiGetSystemTime(*timestamp, "");
+
+  # 2592000 is the epoch time for 30 days, we want to delete items which are older than 30 days in trash from the current date.
+  *month_timestamp = int(*timestamp) - 2592000;
+
+  *KeyValStr = "";
+  msiAddKeyValToMspStr("irodsAdminRmTrash", "", *KeyValStr);
+  foreach(*Row in SELECT META_COLL_ATTR_VALUE, COLL_NAME
+                    WHERE COLL_NAME like '*zone/trash/%'
+                      AND META_COLL_ATTR_NAME like 'ipc::trash_timestamp'
```

Comment:
Since the attribute name is always the same for the trash timestamp, use `=` instead of `like` for comparison.  It will be faster.

Reply:
thank you, done!